api/src/main/java/org/pmiops/workbench/db/dao/UserServiceImpl.java

```diff
@@ -803,20 +799,13 @@ public DbUser syncTwoFactorAuthStatus(DbUser targetUser, Agent agent, boolean is
   }
 
   @Override
-  public DbUser syncDuccVersionStatus(
-      DbUser targetUser, Agent agent, @Nullable Integer signedDuccVersion) {
```

Comment:
OK to remove this param because the `signedDuccVersion` was always pulled from the DbUser anyway